pkg/network/ebpf/c/https.h

```diff
@@ -33,6 +33,8 @@ static __always_inline void https_process(conn_tuple_t *t, void *buffer, size_t
     __builtin_memset(&http, 0, sizeof(http));
     __builtin_memcpy(&http.tup, t, sizeof(conn_tuple_t));
     read_into_buffer((char *)http.request_fragment, buffer, len);
+    http.owned_by_src_port = http.tup.sport;
+    normalize_tuple(&http.tup);
```

Comment:
I _think_ normalize tuple is not necessary here because the SSL ctx to tuple map has the entry already normalized. can you double check that?